web/src/pages/orders/[orderId]/hooks.js

```diff
@@ -1,11 +1,26 @@
+import { orderApi } from '@/hooks/api/orderApi'
 import { useOrder } from '@/hooks/redux/useOrders'
+import { useRouter } from 'next/router'
 
 const useHooks = (orderId) => {
-  const { order, isloading } = useOrder(orderId)
+  const router = useRouter()
+  const { order, isLoading } = useOrder(orderId)
+  const [deleteOrder, isDeleteLoading] = orderApi.useDeleteOrderMutation()
+
+  const handleDelete = async (orderId) => {
```

Comment:
you did not pass any params ni `DeleteModal` so delete it, then it will the orderId of this file 
```suggestion
  const handleDelete = async () => {
```